pkg/sat/sat.go

```diff
@@ -106,6 +106,11 @@ func (r *Resolver) LoadInvolvedPackages(packages []*api.Package) error {
 	}
 	packages = nil
 	for k, _ := range deduplicated {
+		if deduplicated[k].Name == "platform-python" {
+			deduplicated[k].Format.Provides.Entries = append(deduplicated[k].Format.Provides.Entries, api.Entry{
+				Name: "/usr/libexec/platform-python",
+			})
+		}
```

Comment:
Very sophisticated indeed ;)

Is there a way for us not to need to hardcode this hack into bazeldnf? Is the problem that the underlying repo XML doesn't contain this Provides entry? If so, should we talk to the CentOS folks and try to get them to generate the XML so that all information is included? And how can the regular dnf deal with the missing information? Do you think they have a similar hack in there?

Reply:
We can ask, but I think that I simply did not yet understand how modules work on the `rpm` and `dnf` level. I would first want to dive into this a little. I could not find `alternative(pythong)` in provides sections either. My guess would be that `dnf` has some additional knowledge somehow.

For now the workaround should work.

Reply:
Definitely. Will do. :+1:

Reply:
Done.